docs_src/design/ucr/template.md

```diff
@@ -0,0 +1,85 @@
+## Use case template <!-- Replace with use case title -->
+This is a template for EdgeX use cases.
+The description of each section is included as comments and visible only when viewing the [source](https://raw.githubusercontent.com/edgexfoundry/edgex-docs/main/docs_src/design/ucr/template.md).
+Remove this paragraph before submitting a use case.
+
+### Submitters
+<!-- List use case submitters
+
+Format:
+- Name (Organization)
+-->
+
+## Change Log
+<!-- List the changes to the document, incl. state, date, and PR URL.
+State is one of: pending, approved, amended, deprecated
+Date is an ISO 8601 (YYYY-MM-DD) string.
+PR is the pull request that submitted the change, including information such as the diff, contributors, and reviewers.
+
+E.g.:
+- [approved](URL of PR) (2022-04-01)
+- [amended](URL of PR) (2022-05-01)
+-->
+
+### Market Segments
+<!-- What IoT market segments does this use case address? 
+Examples are:
+- Home (home automation, home improvement, energy efficiency)
+- Lifestyle (wearable computing, entertainment and music, family, leisure, pets, toys, drones)
+- Health (fitness, monitoring, measurement, diagnosis, surgery, patience care)
+- Mobility (connected cars, eBikes, aerospace & airports, marine, rail & stations, automotive, traffic)
+- Retail (stores, shops, convenience)
+- Energy (transmission & distribution, fossil, nuclear, alternative)
+- Cities (infrastructure, water & wastewater, HVAC, lighting, security, life safety)
+- Manufacturing (mining, oil & gas, production, supply chain)
+- Public & Services (schools, universities, government, banking, insurance, administration, commercial services)
+- Other (environment, military, agriculture, hospitality)
+
+Reference and more details: https://iot-analytics.com/iot-market-segments-analysis/
+
+Name both the category and industry, e.g.:
+- Retail (stores, convenience)
+-->
+
+### Motivation
+<!-- Summarize the problems that are solved by the use case and why they are important in the relevant domain -->
+
+### Target Users
+<!-- List all stakeholders that are involved in the use case. Examples are:
+- Device Manufacturer
+- Device Owner
+- Device User
+- Device Maintainer
+- Cloud Provider
+- Service Provider
+- Network Operator
+- Software Developer
+- Software Deployer
+- Software Integrator
+-->
+
+### Description
+<!-- Provide the description of the use case from target users' perspective -->
+
+### Existing solutions
+<!-- How is the given use case currently implemented in the industry? List and describe each approach. Highlight possible gaps. -->
+
+### Requirements
+<!-- Provide a list of (non-)functional requirements that aren't addressed by EdgeX with references to corresponding feature request Github issues.
+Include security and privacy requirements that are specific to the given domain.
+
+Format:
+- Feature request issue title ([repo#issue-num](URL))
```

Comment:
IMO, the link to an existing issue should be optional. If issue already created by someone from the community, then it should be referenced here. Otherwise the author can simply document the requirements in this Use Case `Requirements` document.

Reply:
I'm made the following changes to allow listing requirements: https://github.com/edgexfoundry/edgex-docs/pull/775/commits/f68ad8747c9ba50e5d89d7ec6639716c59aff308